pages-functions-cors/package.json

```
@@ -0,0 +1,11 @@
+{
+  "private": true,
+  "version": "0.0.0",
+  "scripts": {
+    "dev": "wrangler pages dev public --local"
```

```suggestion
    "dev": "wrangler pages dev public"
```

Pages doesn't support dev on the edge, so `--local` isn't required.